composer.json

```
@@ -90,7 +70,8 @@
             "cweagans/composer-patches": true,
             "dealerdirect/phpcodesniffer-composer-installer": true,
             "drupal/core-composer-scaffold": true,
-            "phpro/grumphp": true
+            "phpro/grumphp": true,
+            "phpstan/extension-installer": true
```

```suggestion
            "phpro/grumphp": true
```
Are we really `phpstan/extension-installer` use that?

---

I did executions:
`docker-compose exec web composer require drupal/core:~9.4.0 drupal/core-composer-scaffold:~9.4.0 --update-with-all-dependencies --ansi --no-progress --no-interaction`
`docker-compose exec web composer require drupal/core:~10.0 drupal/core-composer-scaffold:~10.0 --update-with-all-dependencies --ansi --no-progress --no-interaction`
with PHP8.1/8.0 and didn't notice that we need that.

---

It's coming from core-dev 10.x, I added it in a previous test where I attempted to add back core-dev. I'll remove it.